pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -32,11 +32,13 @@ var (
 	AWSRestrictedLabelDomains = []string{
 		"k8s.aws",
 	}
-	AMIFamilyBottlerocket = "Bottlerocket"
-	AMIFamilyEKSOptimized = "EKSOptimized"
+	AMIFamilyBottlerocket = "bottlerocket"
```

Comment:
In Kubernetes, YAML values should be capital letters.
Bottlerocket, AL2, Ubunutu

Reply:
Should capacityTypes be uppercase too then?   `[Spot, On-Demand]`?

Reply:
Yes. `OnDemand`, I think.

Reply:
that would be a breaking change right now... so maybe we should just leave it for capacityType? It seems to not be a hard rule values of labels since architecture and operating system label values are lowercase...

Reply:
Hmm this is a good point -- perhaps the difference is whether or not it's a finite enum? Maybe worth asking in sig-api-machinery, or similar.